app/views/collections/show.html.erb

```diff
@@ -1,161 +1,40 @@
 <h1><%= @collection.name %></h1>
 <label>Detail</label>
-<pre style="white-space: pre-wrap; word-break: break-word;"><%= @collection.detail %></pre>
+<%= @collection.detail %>
+<%= link_to 'Edit Collection', edit_collection_path(@collection), class: "btn btn-primary pull-right collection-show-buttons" %>
+<%= link_to 'All Collections', collections_path, class: "btn btn-default pull-right collection-show-buttons" %>
 
-<div class="panel panel-primary">
-  <div class="panel-heading">
-    <h2 class="panel-title">Primary Slide</h2>
-  </div>
+<hr style="border: 1px solid darkgrey;">
+
+<h2>Slides in collection</h2>
   <!-- Table -->
   <table class="table table-striped">
     <thead>
       <tr>
         <th>Slide</th>
         <th>Title</th>
-        <th>Caption</th>
```

Comment:
Captions provide important information for at least the Donor Coverflow kiosk. I don't know if we should hide it